packages/common/core-interfaces/package.json

```diff
@@ -53,13 +53,13 @@
     "typescript-formatter": "7.1.0"
   },
   "typeValidation": {
-    "version": "0.44.1000",
+    "version": "1.1.0",
     "broken": {
-      "RemovedVariableDeclaration_IFluidSerializer": {
+      "RemovedInterfaceDeclaration_IFluidSerializer": {
```

Comment:
Hm this is interesting. @scarlettjlee this should have been removed in fluidframework/core-interfaces@1.0.0 ?

Reply:
You mean we shouldn't need this anymore between 1.0.0 and 1.1.0?

Reply:
Yes. It seems like these two interfaces were already removed in 1.0.0, and the previous reference in our main branch should be "npm:@fluidframework/core-interfaces@^1.0.0".

Reply:
Unsure how I was able to get to the state where CI was passing with this change. Started fresh, and this time around package.json looks clean.